### PR TITLE
docs/improve-datagrouping-docs

### PR DESCRIPTION
**Pull request by pawellysy**
Added demo of performance comparison of a chart with and without dataGrouping.

### CODE REVIEW

**Review by pawelfus — CHANGES_REQUESTED** [review]
List of improvements:
- wrong unit on the yAxis scale (s vs ms)
- missing unit in tooltip
- x-value in the tooltip does not match the format on xAxis (in general, x-value needs info what that number presents)
- add more details:
  - lower = better
  - the machine you used to test it

**Review by bm64 — APPROVED** [review]

**Review by DJTechnoo — APPROVED** [review]

**Review by kamil-musialowski — APPROVED** [review]
Looks good!

**Review by pawelfus — APPROVED** [review]
Thanks!

File: samples/highcharts/blog/dg-performance-comparison/demo.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'spline',
+        zooming: {
+            type: 'xy'
+        }
+    },
+    title: {
+        text: 'chart initial rendering time (lower is better)'
```
Comment (TorsteinHonsi) [inline samples/highcharts/blog/dg-performance-comparison/demo.js:9]:
```suggestion
        text: 'Chart initial rendering time (lower is better)'
```

File: samples/highcharts/blog/dg-performance-comparison/demo.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        },
+        labels: {
+            format: '{(divide value 1000)}k'
+        },
+        categories: [10, 1000, 5000, 10000, 50000, 100000]
+    },
+    tooltip: {
+        shared: true,
+        headerFormat: `<span style="font-size:11px">
+                        chart loading time for {point.x} points:
```
Comment (TorsteinHonsi) [inline samples/highcharts/blog/dg-performance-comparison/demo.js:36]:
```suggestion
                        Chart loading time for {point.x} points: 
```

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks! Just a couple of inline change request on sentence case.

In addition to that, I think it would look better with text and subtitle aligned to the left. Especially on the mobile, long titles don't look so good when centered.

**Review by marvin19 — CHANGES_REQUESTED** [review]
Just two minor suggestions from me:
1. `accessibility.point.valueDescriptionFormat` to give a bit more context to Assitive Technology. Something like this would be good:
`valueDescriptionFormat: '{index}. {xDescription} points, {value} milliseconds.'`
2. The blue text color in the tooltip does not have enough contrast. Consider making the text color a darker blue or make all text black color.A blue color like this: #0179C1 would have sufficient contrast.

**Comment by pawellysy** [comment]
@marvin19 Maybe I'll just skip over to more contrasting colors :D

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**